cmd/req.go

```diff
@@ -471,6 +486,8 @@ func (r *reqCommand) displayResultsAsTXT(res *rpcResults) error {
 	nodeListPrinter(res.Stats.NoResponses, "No Responses from")
 	nodeListPrinter(res.Stats.UnexpectedResponses, "Unexpected Responses from")
 
+	r.w.Flush()
```

Comment:
We should flush in the loop that prints all the results after each result else the UI will pause weirdly till the buffers are full.

Reply:
So end of ddlAssistedPrinter, basicPrinter and errorPrinter should all flush at their end

Reply:
Done